.github/workflows/deploy.yaml

```diff
@@ -1,27 +1,81 @@
-# Deploy tagged versions
-name: Deploy
+# Based on https://github.com/pypa/cibuildwheel/blob/main/examples/github-deploy.yml
+name: Build and deploy to PyPI
 
+# Build on every branch push and pull request
+# Also build (and deploy) when a GitHub Release is created
 on:
   push:
-    tags:
-      - '*'
+  pull_request:
+  release:
+    types:
+      - published
 
 jobs:
-  PyPI-sdist:
-    name: Deploy source distribution to PyPI
-    runs-on: ubuntu-latest
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, windows-2019, macos-10.15]
+
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0  # fetch the entire repo history, required to guarantee versioneer will pick up the tags
+
       - uses: actions/setup-python@v2
+        name: Install Python
         with:
-          python-version: '3.x'
-      - run: pip install twine build
-      - run: python -m build --sdist
-      - run: twine check --strict dist/*
-      - name: "Upload to PyPI"
-        run: twine upload -r pypi dist/pycalphad-*.tar.gz
+          python-version: '3.8'
```

Comment:
Does cibuildwheel use its own Python for building each version? Will py37 and py39 wheels still be built even though py38 is specified here?

Reply:
> Does cibuildwheel use its own Python for building each version?

Correct.

> Will py37 and py39 wheels still be built even though py38 is specified here?

Yes, the py38 is what is used to run cibuildwheel itself. Internally, cibuildwheel has it's own build matrix of platforms x Python versions where it does isolated installs of the different Python versions and builds the wheels accordingly.